ra2ce/analysis/losses/losses_base.py

```diff
@@ -104,17 +82,15 @@ def __init__(
         self.production_loss_per_capita_per_hour = (
             self.analysis.production_loss_per_capita_per_hour
         )
+
         self._check_validity_analysis_files()
-        self.intensities = _load_df_from_csv(
-            Path(self.analysis.traffic_intensities_file), [], self.link_id
-        )  # per day
+        self.intensities = TrafficIntensitiesReader(self.link_id).read(
+            self.analysis.traffic_intensities_file
```

Comment:
This could be a explicit file (even mapped to the reader directly at the `init`). This applies to `values_of_time_file` as well.

Reply:
What do you mean with "explicit"?

Reply:
Confusing regarding the scope. As we're in the init still leave it like this.